### ISSUE TITLE
[a11y walker] enable inline math

### ISSUE THREAD

**Issue by pkra** [mathjax/MathJax-RespEq#66]
Currently, all extensions skip inline math. That makes sense for responsive rendering (though I might argue for that there, too). But for the a11y walker we need to enable inline math elements.

**Comment by dpvc** [comment]
The `Semantic-Collapse` extension only collapses displayed equations, but `Semantic-MathML` should process all math, and I don't see anything in the walker that would exclude in-line math.  In fact, I've just verified that the walked *does* walk in-line math.

If you could post the example file where this isn't working for you, that might help.

**Comment by pkra** [comment]
Hm. Maybe I was just sleep deprived after the deadline. I'll check.

**Comment by zorkow** [comment]
@dpvc I am trying to understand why the inline math expressions do not get the data-semantic attributes attached and am a bit a loss. I am working with the Struik example.

They are being enriched correctly, with jax.enriched  set to the enriched MathML. They also get the correct events added, but the data-semantic attributes are not pushed through, so we can not speak or walk. Is there another round of rerendering missing?

**Comment by zorkow** [comment]
FWIW, I've run reprocess() but that does not help either.

**Comment by pkra** [comment]
> FWIW, I've run reprocess() but that does not help either.

Have you tried rerender? IIRC, reprocess re-digests the input which (IIUC) wouldn't have changed.

**Comment by zorkow** [comment]
Tried both.

**Comment by pkra** [comment]
Ah, I should've figured...

**Comment by dpvc** [comment]
It looks like some of the Semantic-Collapse code needs to be refactored into Semantic-MathML, and that may mean that the pipe-line between the two needs improvement.  I'll have to see what can be done.

As a temporary work-around, you could probably add

```
}
if (jax.enriched) {
```

right after [line 156](https://github.com/mathjax/MathJax-RespEq/blob/master/extensions/Semantic-Collapse.js#L156) in Semantic-Collapse.js so that the enriched form is always turned into the internal form (this was only being done for display equations, as Peter pointed out; not sure what walking I was doing, perhaps by syntax rather than semantics; sorry about that).

**Comment by zorkow** [comment]
PR #77 contains the work around.

**Comment by dpvc** [comment]
PR #79 should resolve this issue.

**Comment by zorkow** [comment]
Done.
